### PR TITLE
Feature/app icon resumption

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
Fixes #[1456](https://github.com/smartdevicelink/sdl_core/issues/1456)

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Testing Plan
Test scripts PR : https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1867
HMI PR : https://github.com/smartdevicelink/sdl_hmi/pull/63

### Summary
This PR contains the implementation of proposal https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0041-appicon-resumption.md 

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: app/model/sdl/Abstract/Model.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-/*
- * Copyright (c) 2013, Ford Motor Company All rights reserved.
+/* * Copyright (c) 2013, Ford Motor Company All rights reserved.
```
[the remark this answers is not in the document]
Comment (AKalinich-Luxoft) [reply app/model/sdl/Abstract/Model.js]: @jacobkeeler yes, thanks.
Fixed in https://github.com/smartdevicelink/sdl_hmi/pull/63/commits/1e55a6edfb168500a9c34315bdf8f239302c3e58

File: app/controller/sdl/Abstract/Controller.js [hunk 2]
```diff
@@ -837,78 +837,7 @@ SDL.SDLController = Em.Object.extend(
...
-            'position': 0
-          },
-          cmdID: -3
-        }
-      };
-      SDL.SDLController.getApplicationModel(params.appID).addCommand(
-        exitCommand
-      );
-    },
+
```
[the remark this answers is not in the document]
Comment (AKalinich-Luxoft) [reply app/controller/sdl/Abstract/Controller.js]: @jacobkeeler reverted in https://github.com/smartdevicelink/sdl_hmi/pull/63/commits/1e55a6edfb168500a9c34315bdf8f239302c3e58

File: app/model/sdl/RModel.js [hunk 3]
```diff
@@ -187,6 +211,84 @@ SDL.RModel = SDL.SDLModel.extend({
...
+        cmdID: -3
+      }
+    }
+    SDL.SDLController.getApplicationModel(params.appID).addCommand(
+      exitCommand
+    );
+  },
+  applicationModels: {
+    0: SDL.SDLMediaModel,
+    1: SDL.SDLNonMediaModel
```
[the remark this answers is not in the document]
Comment (AKalinich-Luxoft) [reply app/model/sdl/RModel.js]: @jacobkeeler reverted in https://github.com/smartdevicelink/sdl_hmi/pull/63/commits/1e55a6edfb168500a9c34315bdf8f239302c3e58

**Comment by JackLivio** [comment]
@LuxoftAKutsan @AKalinich-Luxoft Please review merge conflict with develop template images changes. Thank you!

**Comment by AKalinich-Luxoft** [comment]
@JackLivio merge conflicts have been resolved

**Pull request merged**